docs/getting-started/first-release.md

```diff
@@ -0,0 +1,199 @@
+---
+template: main.html
+---
+
+# Your first blue-green release
+
+This tutorial shows how Iter8 can be used to release a basic Kubernetes application using a blue-green rollout strategy. The user declaratively describes the desired application state at any given moment. An Iter8 `release` chart assists users who describe the application state at any given moment. The chart provides the configuration needed for Iter8 to automatically deploy application versions and configure the routing to implement a blue-green rollout strategy.
+
+In a blue-green rollout, a percentage of requests are directed to a candidate version of the model. This percentage can be changed over time.
+
+![Blue-green rollout](../tutorials/images/blue-green.png)
+
+In this tutorial, we use the Istio service mesh to distribute inference requests between different versions of a model.
+
+???+ warning "Before you begin"
+    1. Ensure that you have a Kubernetes cluster and the [`kubectl`](https://kubernetes.io/docs/reference/kubectl/) and [`helm`](https://helm.sh/) CLIs. You can create a local Kubernetes cluster using tools like [Kind](https://kind.sigs.k8s.io/) or [Minikube](https://minikube.sigs.k8s.io/docs/).
+    2. Install [Istio](https://istio.io). It suffices to install the [demo profile](https://istio.io/latest/docs/setup/getting-started/), for example by using: 
+    ```shell
+    istioctl install --set profile=demo -y
+    ```
+
+## Install the Iter8 controller
+
+--8<-- "docs/getting-started/install.md"
+
+```shell
+export IMG=kalantar/iter8:20231011-1506
+export CHARTS=/Users/kalantar/projects/go.workspace/src/github.com/iter8-tools/iter8/charts
+helm upgrade --install iter8 $CHARTS/controller \
+--set image=$IMG --set logLevel=trace \
+--set clusterScoped=true
+```
+
+## Deploy initial version
+
+Deploy the initial version of the application using the Iter8 `release` chart by identifying the environment into which it should be deployed, a list of the versions to be deployed (here just one), and the rollout strategy to be used:
+
+```shell
+cat <<EOF | helm upgrade --install httpbin $CHARTS/release -f -
+environment: deployment-istio
+application: 
+  versions:
+  - metadata:
+      labels:
+        app.kubernetes.io/version: v0
+    image: kennethreitz/httpbin
+  strategy: blue-green
+EOF
+```
+
+??? note "What happens?"
+    Because `environment` is set to `deployment-istio`, a `Deployment` and a `Service` object are created.
+        - The namespace `default` is inherited from the helm release namespace since it is not specified in the version or in `application.metadata`.
```

Comment:
```suggestion
        - The namespace `default` is inherited from the Helm release namespace since it is not specified in the version or in `application.metadata`.
```